thunder/tests/framework.py

```diff
@@ -55,6 +55,7 @@ class NOTHING:
 DISABLE_CUDA_TEST_INSTANTIATION: bool = (
     env_var_DISABLE_CUDA_TEST_INSTANTIATION == "true" or env_var_DISABLE_CUDA_TEST_INSTANTIATION == "1"
 )
+IS_WINDOWS = os.name == "nt"
```

Comment:
super minor nit: maybe `platform.system() == "Windows"` is a bit clearer? Did not know what NT stands for :)